ts/screens/authentication/IdpSelectionScreen.tsx

```diff
@@ -33,10 +33,19 @@ import { H1 } from "../../components/core/typography/H1";
 import ButtonDefaultOpacity from "../../components/ButtonDefaultOpacity";
 import { IOStyles } from "../../components/core/variables/IOStyles";
 import { VSpacer } from "../../components/core/spacer/Spacer";
+import { IdpData } from "../../../definitions/content/IdpData";
 
 type Props = ReturnType<typeof mapStateToProps> &
   ReturnType<typeof mapDispatchToProps>;
 
+const TestIdp: SpidIdp = {
```

Comment:
I would prefer to leave it there, as it is done for the IdpCIE in the LandingScreen, since both of them are just used inside the respective files and I do not see a necessity of accessing them (outside of their respective files) in the foreseeable future